### PR TITLE
chat action

### PR DESCRIPTION
**Pull request by Sunda001**
new chat action for GameScore

### CODE REVIEW

File: telethon/events/chataction.py [hunk 1]
```diff
@@ -113,7 +114,11 @@ def build(cls, update, others=None, self_id=None):
             elif isinstance(action, types.MessageActionPinMessage) and msg.reply_to:
                 return cls.Event(msg,
                                  pin_ids=[msg.reply_to_msg_id])
-
+            elif isinstance(action, types.MessageActionGameScore):
+                return cls.Event(msg,
+                                 game_id=action.game_id,
```
Comment (Lonami) [inline telethon/events/chataction.py:119]: The `Event` class does not have this parameter. Doesn't this fail?

**Review by Lonami — CHANGES_REQUESTED** [review]

File: telethon/events/chataction.py (hunk 1 above)
Comment (Sunda001) [reply telethon/events/chataction.py]: ah yes done

**Review by Lonami — CHANGES_REQUESTED** [review]
The Members docstring of the Event also needs to be updated to document the new member. Also, is there no way to get the Game object? Knowing just the score value is not very useful.

**Comment by Sunda001** [comment]
yeah you can get game obj from attribute reply_to_msg_id

**Comment by Lonami** [comment]
Could we also add a `async def get_game`? Along with an attribute to get the game ID.

**Pull request merged**